components/scream/src/mct_coupling/scream_scorpio_interface.F90

```diff
@@ -629,6 +712,58 @@ subroutine get_decomp(tag,dtype,dimension_len,compdof,iodesc)
     end if 
 
   end subroutine get_decomp
+!=====================================================================!
+  subroutine set_dof(filename,varname,dof_len,dof_vec)
```

Comment:
I can't figure out what dof means here. I usually think "degrees of freedom". Maybe add a comment or make the function name more descriptive?

Reply:
@PeterCaldwell , good suggestion.  I've added comments to every routine in scream_scorpio_interface.F90.